feature/switcher

Olha-buz opened the pull request:
обговорити світчер, ще все сире

Olha-buz committed:
feature/switcher

обговорити світчер, ще все сире

AlinaTantsura commented on src/js/dark-light-switcher.js:
```diff
@@ -0,0 +1,43 @@
+                logo.classList.toggle('hidden-header')
+            } else {
+                logo.classList.toggle('hidden-header');
+            }
```
Продивись логіку операції, бо перевіряєш умову, і робиш якуйсь дію і коли умова правдива і коли ні, а робиш одну й ту саму операцію

AlinaTantsura reviewed (changes requested):
Розбирайся далі. З іконкою букшелф я розберусь.

Olha-buz committed:
fix switcher

залишилось доробити, щоб при перезагрузки сторінки колір був той, на який останній перемикали, та при перемиканні між сторінками тема не змінювалась

AlinaTantsura committed:
Merge branch 'main' into feature/switcher
